test/functional/adb-commands-e2e-specs.js

```diff
@@ -243,8 +243,8 @@ describe('adb commands', function () {
     });
 
     it('should return the report as base64-encoded .zip archive if needed', async function () {
-      if (await adb.getApiLevel() < 23) {
-        // This option does not work with older APIs
+      if (process.env.TRAVIS) {
```

Comment:
I do not think that this is related to tools.

```adb bugreport myreport.zip```

Emu API 23 output:
>Failed to get bugreportz version: 'bugreportz -v' returned '/system/bin/sh: bugreportz: not found' (code 0).
If the device does not run Android 7.0 or above, try 'adb bugreport' instead.

Real Device API 21&22:
>Bugreport is in progress and it could take minutes to complete.
Please be patient and do not cancel or disconnect your device until it completes.
Unexpected string (/system/bin/sh: bugreportz: not found) returned by bugreportz, device probably does not support -z option